gitpod-io/gitpod
# Add CDE Universe footnote

## Description

This will add a footnote on the workspace start page with a link to the CDE Universe.

## How to test
Open a new workspace and notice the footnote on the workspace start, see bottom right.

#### Preview status

gitpod:summary

## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold
